tests/Type/WebMozartAssert/data/type.php

```diff
@@ -231,46 +232,46 @@ public function isInstanceOfAny($a, $b, $c, $d, $e, $f, $g): void
 	public function notInstanceOf($a, $b, $c): void
 	{
 		Assert::notInstanceOf($a, Bar::class);
-		\PHPStan\Testing\assertType('PHPStan\Type\WebMozartAssert\Foo', $a);
+		assertType('PHPStan\Type\WebMozartAssert\Foo', $a);
 
 		Assert::notInstanceOf($b, new stdClass());
-		\PHPStan\Testing\assertType('PHPStan\Type\WebMozartAssert\Foo', $b);
+		assertType('PHPStan\Type\WebMozartAssert\Foo', $b);
 
 		Assert::notInstanceOf($c, 17);
-		\PHPStan\Testing\assertType('PHPStan\Type\WebMozartAssert\Bar|PHPStan\Type\WebMozartAssert\Foo', $c);
+		assertType('PHPStan\Type\WebMozartAssert\Bar|PHPStan\Type\WebMozartAssert\Foo', $c);
 	}
 
 	public function isAOf($a, $b, string $c): void
 	{
 		Assert::isAOf($a, stdClass::class);
-		\PHPStan\Testing\assertType('stdClass', $a);
+		assertType('stdClass', $a);
 
 		Assert::isAOf(Foo::class, stdClass::class);
-		\PHPStan\Testing\assertType('*NEVER*', Foo::class);
+		assertType('*NEVER*', Foo::class);
 
 		Assert::isAOf(Bar::class, stdClass::class);
-		\PHPStan\Testing\assertType('\'PHPStan\\\Type\\\WebMozartAssert\\\Bar\'', Bar::class);
+		assertType('\'PHPStan\\\Type\\\WebMozartAssert\\\Bar\'', Bar::class);
 
 		Assert::nullOrIsAOf($b, stdClass::class);
-		\PHPStan\Testing\assertType('stdClass|null', $b);
+		assertType('stdClass|null', $b);
 
 		Assert::isAOf($c, stdClass::class);
-		\PHPStan\Testing\assertType('class-string<stdClass>', $c);
+		assertType('class-string<stdClass>', $c);
 	}
 
 	public function isAnyOf($a, $b): void
 	{
 		Assert::isAnyOf($a, [DateTimeImmutable::class, stdClass::class]);
-		\PHPStan\Testing\assertTyp0e('DateTimeImmutable|stdClass', $a);
+		assertType('DateTimeImmutable|stdClass', $a);
```

Comment:
look at that, hidden typo that disabled the assertion I suppose..

Reply:
Cool :)